presto-docs/src/main/sphinx/release/release-331.rst

```diff
@@ -0,0 +1,83 @@
+=========================
+Release 331 (XX Feb 2020)
+=========================
+
+General Changes
+---------------
+
+* Prevent query failures when worker is shut down gracefully. (:issue:`2648`)
+* Fix join failures for queries involving OR predicate with non comparable functions. (:issue:`2861`)
+* Ensure query completed event is fired when there is an error during analysis or planning. (:issue:`2842`)
+* Fix query memory accounting for ``ORDER BY`` queries. (:issue:`2612`)
+* Fix :func:`last_day_of_month` for ``timestamp with time zone`` values. (:issue:`2851`)
+* Fix excessive runtime when parsing deeply nested expressions with unmatched parenthesis. (:issue:`2968`)
+* Correctly reject ``date`` literals that cannot be represented in Presto. (:issue:`2888`)
+* Improve query performance by removing redundant data reshuffling. (:issue:`2853`)
+* Improve performance of inequality joins involving ``BETWEEN``. (:issue:`2859`)
+* Improve engine performance in joins. (:issue:`2862`)
+* Enable dynamic filtering by default. (:issue:`2793`)
+* Show reorder join cost in ``EXPLAIN ANALYZE VERBOSE`` (:issue:`2725`)
+* Allow configuring resource groups selection based on user's groups. (:issue:`3023`)
+
+Security Changes
+----------------
+
+* Remove unused ``internal-communication.jwt.enabled`` configuration property. (:issue:`2709`)
+* Rename JWT configuration properties from ``http.authentication.jwt.*`` to ``http-server.authentication.jwt``. (:issue:`2712`)
+* Add access control checks for query execution, view query, and kill query. This can be configured using
```

Comment:
```
* Add access control checks for query execution, view query, and kill query. This can be
  configured using :ref:`query_rules` in :ref:`file-based-system-access-control`. (:issue:`2213`)